Tools/scripts/build_binaries.py

```diff
@@ -62,21 +84,36 @@ def board_options(self, board):
             return ["--static"]
         return []
 
-    def run_waf(self, args):
+    def change_compiler(self, versiontag):
+        '''change compiler using gcc-version.sh script'''
+        self.run_program("BB-Compiler", ["gcc-version.sh", versiontag])
+
+    def run_waf(self, args, compiler=None):
         if os.path.exists("waf"):
             waf = "./waf"
         else:
             waf = os.path.join(".", "modules", "waf", "waf-light")
         cmd_list = [waf]
         cmd_list.extend(args)
-        self.run_program("BB-WAF", cmd_list)
+        env = None
+        if compiler is not None:
+            gcc_path = os.path.join(os.environ["HOME"], "arm-gcc", compiler, "bin")
```

Comment:
```suggestion
            gcc_path = os.path.join(os.environ["HOME"], "gcc-arm", compiler, "bin")
```

surely!

if not, or possibly as well: 
```suggestion
            gcc_home = os.environ("AP_GCC_HOME", os.path.join(os.environ["HOME"], "arm-gcc"))
            gcc_path = os.path.join(gcc_home, compiler, "bin")
```

just so others don't have to have the same directory structure :-)

Reply:
good idea, added